math/classical_sieve_of_eratosthenes.cpp

```diff
@@ -39,8 +38,10 @@ void print(int N) {
 }
 
 int main() {
+  fastio();
   int N = 100;
-  sieve(N);
-  print(N);
+  bool* isprime = new bool[N];
```

Comment:
let it be `bool *isPrime = new bool[N];` or `bool *is_prime = new bool[N];`

Reply:
@ayaankhan98 I think name `isprime` looks good because in other `sieve_of_eratosthenes.cpp` file it has also the same name `isprime`.

Reply:
@ayaankhan98 I think name `isprime` looks good because in other `sieve_of_eratosthenes.cpp` file it has the same name `isprime` so it maintains the consistency.

Reply:
if the algorithm is already present, then i think there is no need of the same stuff what you can do is that you can improve the old file if you see some mistake or you can optimize the previously written algorithms.

Reply:
@ayaankhan98 kindly check discussion on #820

Reply:
@iamrajiv oh okay so this is the optimised version of algorithm which is already present, that's what i was suggesting also to optimize the already present algorithm

Reply:
@ayaankhan98 Initially I was doing the same i.e. optimising the existing algorithms but @kvedala and @mishraabhinn suggested to submit a different implementation for an existing algorithm and I agree with them because this repository is for educational purposes and I think people should know different approaches to solve the same problem and that what is has been done in `sieve_of_eratosthenes` since the previous code was not been implemented using Sieving till root but this code uses Sieving till root approach so it makes people learn a new approach.